src/schema/v2/me/__tests__/collectionsConnection.test.ts

```diff
@@ -43,6 +48,39 @@ describe("collectionsConnection", () => {
       offset: 0,
       size: 1,
       total_count: true,
+      artwork_id: "123-abc",
+    })
+  })
+
+  it("passes correct args to Gravity when `includesArtworkID` arg is specified", async () => {
+    const query = gql`
```

Comment:
We just need to remove this test case now, since it's already covered above 😄

Reply:
Addressed here: https://github.com/artsy/metaphysics/pull/5022/commits/4bb9e4d4861a1434b82f15da72fd40d6112803cb